lib/spack/spack/build_systems/autotools.py

```diff
@@ -16,7 +16,7 @@
 from subprocess import check_call
 
 import llnl.util.tty as tty
-from llnl.util.filesystem import working_dir, force_remove
+from llnl.util.filesystem import working_dir, force_remove, find
```

Comment:
I would prefer to keep, at least for newly introduced functions, the module namespace:
```python
import llnl.util.filesystem as fs
```
and use `fs.find` later on.

Reply:
I fixed it. I referred to this.
https://github.com/spack/spack/pull/18850/